internal/servers/controller/handlers/authmethods/password.go

```diff
@@ -140,6 +166,34 @@ func (s Service) authenticateWithPwRepo(ctx context.Context, scopeId, authMethod
 	return prot, nil
 }
 
+func validateAuthenticatePasswordRequest(req *pbs.AuthenticateRequest) error {
+	badFields := make(map[string]string)
+
+	attrs := req.GetAttributes().GetFields()
+	if _, ok := attrs[loginNameField]; !ok {
```

Comment:
it might be nice to define consts for these keys.

Reply:
Going to defer to some other time